package.json

```diff
@@ -86,11 +85,12 @@
     "npm": "^5.8.0",
     "plist": "^2.0.1",
     "prop-types": "^15.6.1",
-    "react": "^15.3.1",
-    "react-dom": "^15.3.1",
-    "react-redux": "^5.0.2",
-    "redux": "^3.5.2",
-    "redux-saga": "^0.14.3",
+    "react": "16.3.2",
+    "react-dom": "16.3.2",
+    "react-redux": "5.0.7",
+    "react-test-renderer": "16.3.2",
+    "redux": "3.7.2",
```

Comment:
Is `redux` + redux friends necessary for the `react` upgrade. If so, so be it since they'll be gone anyways.

Reply:
it's better than having old version dangling around for now :)